K2Bridge/KustoDAL/KustoResponseParser.cs

```diff
@@ -173,6 +173,12 @@ private void ReportQueryExecutionMetrics(KustoResponseDataSet kustoResponseDataS
                 if (string.IsNullOrWhiteSpace(aggregationType))
                 {
                     // This is not a bucket aggregation scenario
+                    if (dataRowCollection.Count == 0)
+                    {
+                        var table = kustoResponse[KustoTableNames.Aggregation].TableData;
+                        dataRowCollection.Add(table.NewRow());
```

Comment:
Yes. In some situation, table returns columns but no rows (for instance for standalone aggregation out of ranges). AddAggregates expect to read columns name if the value is empty or NaN (for instance for percentiles). But, if rows are empty, we don't enter in AddAggregates. I currently fixed this issue by adding an Empty Row, but it perhaps more appropriate to pass the Table to AddAggregates and rework the logic. I suggest to keep this current fix and open a parallel discussion / workItem to check if there is a more appropriate way to address this point.

Reply:
I guess my question is, if we actually got a table that has just one empty row, will it break/change something?

I don't think it would, but I want to be sure.

Reply:
No, I ensure it does not break anything. I tested this hack and it offers only benefits.